app/hallebarde/infrastructure/file_repository.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+import boto3
+import hallebarde.config
+
+BUCKET_NAME = f'hallebarde-storage-{hallebarde.config.ENVIRONMENT}'
+
+
+def get_files_list(identifier: str) -> List:
```

Comment:
Set get_files_list private and add a get_file(identifier) method which returns only the last added file